### ISSUE TITLE
Unable to use/open asset in Unity, error CS0106

### ISSUE THREAD

**Issue by Katyusha117** [vrm-c/UniVRM#2107]
Hello! I'm very new to github and not used to how everything is laid out here, so I'm just going to fill in the information that is asked for below ^^:

## Environments (please complete the following information):

 - UniVRM version: VRM-0.112.0_6a46
 - Unity version: `Unity-2019.4.31f1`
 - OS: `Windows 10`

## Describe the bug

* A clear and concise description of what the bug is: I've imported the package into Unity, but it shows an error CS0106 and I can't access the VRM tab at all.
* ConsoleLog: Assets\VRM10\Runtime\Components\Constraint\IVrm10Constraint.cs(7,23): error CS0106: The modifier 'internal' is not valid for this item
* ScreenShot: Not needed, error message provided (if you do need one of something, let me know)
* Reproduction procedure: I started a new project, imported the same VRM package and still received the same error.
* File that causes an error (those that can be pasted here): It won't let me paste it, but it's the "IVrm10Constraint.cs" file displayed in the error message.

I opened the file and attempted to change the "internal" modifier, but as I'm not used to C# or really programming in general, I wasn't sure what I'd need to change it to ^^;

**Comment by Quaxdrone** [comment]
You Need Upgrade Unity to 2021 version(2019 Version is not Supported)

**Comment by Katyusha117** [comment]
Okay, I've opened a new project in a newer version of Unity (2020.3.27f1) and I was able to load VRM.
But I was following a tutorial to convert my avatar and something seems to have not gone as it was supposed to.
When I exported the model and brought it back into unity, it showed that a bunch of folders were supposed to be made and other thing automatically assigned. None of the folders were created and I'm not sure what to really do with anything at the moment 😓
